frog/imports/client/Wiki/index.js

```diff
@@ -74,7 +80,8 @@ type WikiCompStateT = {
 };
 
 class WikiComp extends React.Component<WikiCompPropsT, WikiCompStateT> {
-  wikiId: string = this.props.match.params.wikiId;
+  wikiId: string =
+    this.props.match?.params.wikiId || this.props.embedPage.wikiId;
```

Comment:
embedPage is type Maybe, what happens if it is undefined?

Reply:
Which would mean it is not an embedded page but called by the `react-router`.

Reply:
Yes, but what happens if they are both undefined, which is a possible state

Reply:
It is possible but should never happen (you would either access the wiki via `react-router` or embed it, no other possible way to use it).